### ISSUE TITLE
Kernel 4.14 support

### ISSUE THREAD

**Issue by b0x1** [sophos/talpa#7]
The Sophos AV on-access scanning doens't run on my Fedora with kernel 4.14 even though the Talpa module compiles and even when SELinux  is switched off. 

```
# /opt/sophos-av/engine/talpa_select select
[Talpa-select]
Copyright 1989-2017 Sophos Limited. All rights reserved.
Wed Jan 3 10:41:14 2018 GMT
Linux distribution: [fedora]
Product: [Fedora release 27 (Twenty Seven)]
Kernel: [4.14.8-300.fc27.x86_64]
Multiprocessor support enabled.
Searching for source pack...
Searching for suitable binary pack...
Binary pack was created locally.
Found suitable binary pack. Using: /opt/sophos-av/talpa/compiled/talpa-binpack-fedora-x86_64-4.14.8-300.fc27.x86_64-1smpweddec20190018utc2017.tar.gz
```

And yet:

```
$ sudo /opt/sophos-av/bin/savdstatus
Sophos Linux Security is active but on-access scanning is not running
```

**Comment by paperclip** [comment]
Yes, kernel 4.14 removed the vfs_read/vfs_write symbols, but not the header entries, so Talpa compiles but can't load.
We've pushed updates that should fix talpa for 4.14 and 4.15.

**Comment by paperclip** [comment]
You can use https://gist.github.com/paperclip/7892031 to get talpa from github to SLS.
